pkg/operator/controller.go

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
What was the rationale behind moving pkg/controller to pkg/operator

Reply:
This is the convention in other operators, mostly because operators grow to have more than one controller in them and each controller is organized in a new subdirectory of `pkg/operator`.
https://github.com/openshift/cluster-kube-apiserver-operator/tree/master/pkg/operator